### ISSUE TITLE
An idea

### ISSUE THREAD

**Issue by mikesimarta** [kotarou3/UNSW-Course-Dependency-Viewer#1]
Hi, this may be something hard to implement. Can you show future courses dependencies? (For example, I typed in COMP1927, the prereq is [COMP1911 + bridging, COMP1917 or COMP1921] and the courses which has a prereq of COMP1927 are [COMP2911 and ...] and the courses which has a prereq of COMP2911 are ... And so on

However good job on the site

**Comment by kotarou3** [comment]
You mean something like reverse dependencies?

**Comment by mikesimarta** [comment]
yes and you can have an option to hide future courses if they don't want to see it. (It's similar to how Pathways show 1 level in the future, but you can do many).

**Comment by nickw444** [comment]
+1
